daprdocs/content/en/operations/troubleshooting/common_issues.md

```diff
@@ -226,3 +226,24 @@ In order for mDNS to function properly, ensure `Micorosft Content Filter` is ina
 Microsoft Content Filter is disabled when the output is "Success".
 
 > Some organizations will re-enable the filter from time to time. If you repeatedly encounter app-id values missing, first check to see if the filter has been re-enabled before doing more extensive troubleshooting.
+
+## admission webhook denied the request
```

Comment:
@daixiang0, can you please elaborate on the scenario where this error would occur? Similarly to the other examples in this doc that provide explanations on what the user is trying to accomplish and what the reason for the error is. Thank you

Reply:
Update.